_posts/2021-03-12-automated-accessibility-testing-leveraging-github-actions-and-pa11y-ci-with-axe.md

```diff
@@ -99,9 +99,9 @@ Running Pa11y on 49 URLs:
 ✔ 49/49 URLs passed
 ```
 
-As you can see, at the time of this writing we have all our pages passing! To see examples of us fixing reported accessibility issues check out [https://github.com/CivicActions/accessibility/pull/128](https://github.com/CivicActions/accessibility/pull/128).
+As you can see, at the time of this writing we have all our pages passing! To see examples of us fixing reported accessibility issues check out [pull request #128](https://github.com/CivicActions/accessibility/pull/128).
```

Comment:
For consistency both this and the next link probably should be:
- pull request #128 on accessibility site
- pull request #128 on pa11y-ci repo

Reply:
The second one is an issue. How about this for link text:

* `pull request #128 on this site`
* `issue #128 on the pa11y-ci repository`

Reply:
That's just great.